src/node_mgmt/commands.py

```diff
@@ -211,6 +218,71 @@ def readall(s):
     return result
 
 
+def holykell_sensor_address_7(node):
+    return _change_address_holykell(target_slave_id=7)
+
+
+def holykell_sensor_address_8(node):
+    return _change_address_holykell(target_slave_id=8)
+
+
+def _change_address_holykell(target_slave_id: int) -> dict:
+    mod = minimalmodbus.Instrument(port=DEFAULT_SERIAL_DEV, slaveaddress=1, debug=True)
+    mod.serial.baudrate = DEFAULT_SERIAL_BAUD_RATE
+    mod.serial.timeout = 3
+    result = {}
+
+    # confirm first that Holykell is accessible on default slave id 1
+    try:
+        logger.info('Checking if communication with holykell is up')
```

Comment:
Do you think this is necessary? If communication is down, then `_set_address_holykell()` will itself simply fail - so I'm not sure there's much point in verifying communication before we carry out the address change.

The "Pythonic" way of doing things is to simply try doing what you want to do, and "ask for forgiveness" if it doesn't work :)

Reply:
Good point, I wanted to separate the error case where communication is just down from the error case where something went wrong afterwards while assigning the new address. Though my guess is the latter one would be pretty rare and it isn't really worth differentiating the two. Correcting that as well.